hangman.c

```diff
@@ -0,0 +1,116 @@
+// Julia Tan
+#include <stdio.h>
+#include <string.h>
+
+int main(){
```

Comment:
Your code doesn't necessarily handle capitals, but that's fine since it doesn't say to do so in the instructions. Besides that, nice job! (and great improvement on commenting your code)